src/compiler/diagnosticMessages.json

```diff
@@ -3735,6 +3735,14 @@
         "category": "Error",
         "code": 2866
     },
+    "Cannot find name '{0}'. Do you need to install type definitions for bun? Try `npm i --save-dev @types/bun`.": {

```

Comment:
Could we capitalize Bun when used in prose?

Reply:
That is entirely up to you all, I assumed lowercase but I do see that the website capitalizes it consistently, so I can fix it here.

Reply:
Cheers, thanks.